doc/_static/custom_scripts.js

```diff
@@ -0,0 +1,162 @@
+            popover: {
+              title: "Status Bar",
+              description: 'The status bar shows your current Python environment, git branch, memory usage and various attributes of your current file.',
+              position: "top",
+            },
+          },
+          {
+            element: "#options-menu-rect",
+            popover: {
+              title: "Options Menu",
+              description: 'You can display the options menu by clicking the hamburger icon at the top right of the pane. It contains useful options and actions relevant to the pane.',
+              position: "right",
+            },
+          },
+          {
+            element: "#context-menu-rect",
+            popover: {
+              title: "Context Menu",
+              description: 'To display the context menu for a pane, right-click anywhere over it. The menu shows actions relevant to the element you are hovering over.',
+              position: "right",
+            },
+          },
+          {
+            element: "#editor-rect",
+            popover: {
+              title: "Editor",
+              description: 'The <a href="editor.html">Editor</a> is the pane where you can create, open and edit files. It contains useful features like autocompletion, real-time analysis and syntax highlighting.',
+              position: "right",
+            },
+          },
+          {
+            element: "#console-rect",
+            popover: {
+              title: "Ipython Console",
+              description: 'The <a href="ipythonconsole.html">Console</a> allows you to run your code from the Editor or interactively. You can also use it to interact with Spyder’s debugger.',
+              position: "left",
+            },
+          },
+          {
+            element: "#help-rect",
+            popover: {
+              title: "Help",
+              description: 'The <a href="help.html">Help</a> pane displays documentation for the objects you are using in the Editor or the IPython Console. To use it, press Ctrl-I (Cmd-I on macOS) with your cursor over an object, or type its name in the object field.',
+              position: "left",
+            },
+          },
+          {
+            element: "#variable-explorer-rect",
+            popover: {
+              title: "Variable Explorer",
+              description: 'The <a href="variableexplorer.html">Variable Explorer</a> allows you to browse and interact with the objects generated running your code. Double-clicking a variable will open a specialized viewer, allowing you to inspect its contents.',
+              position: "left",
+            },
+          },
+          {
+            element: "#plots-rect",
+            popover: {
+              title: "Plots",
+              description: 'The <a href="plots.html">Plots</a> pane shows the figures and images created during your code execution. It allows you to browse, zoom, copy, and save the generated plots.',
```

Comment:
```suggestion
              description: 'The <a href="plots.html">Plots</a> pane shows the figures and images created by your code. It allows you to browse, zoom, copy, and save the generated plots.',
```

How about this? Its simpler and it addresses your concern that the plots are created by the code, not directly by the user

Reply:
If you don't like it thought its fine, up to you.